justfile

```diff
@@ -0,0 +1,284 @@
+# Configuration
+# set shell := ["bash", "-uc"]
+
+# Environment Variables
+export BASH_SILENCE_DEPRECATION_WARNING := "1"
+export CLICOLOR_FORCE := "1"  # For `tree`
+export LESS := "-FRXi"        # F: no-op single page, R: color, X: keep text when exiting, i: case insensitive
+export PACKAGES := "bind9-host curl fping git less tmux tree"
+
+# Get operating system
+OS := `uname -s`
+
+# Default recipe (shows available recipes)
+default:
+    @just --list
+
+# Development Environment Setup
+[private]
+_ensure-homebrew:
+    #!/usr/bin/env bash
+    if [[ "{{OS}}" == "Darwin" && ! -x $(command -v brew) ]]; then
+        [[ -d /opt/homebrew/bin ]] && export PATH="/opt/homebrew/bin:$PATH"
+        [[ -x $(command -v brew) ]] || echo "ERROR: homebrew missing"
+    fi
+
+# Activate virtual environment after changing directory
+a:
+    #!/usr/bin/env bash
+    echo $SHELL
```

Comment:
Maybe print the version too to match `pathver`? I've developer a tweaked sed line that can handle bash too.

```suggestion
    echo $SHELL $BASH_VERSION
```